sig-auth/charter.md

```diff
@@ -0,0 +1,68 @@
+# SIG Auth Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+SIG Auth is responsible for the design, implementation, and maintenance of features in
+Kubernetes that control and protect access to the API and other core components. This includes
+authentication and authorization, but also encompasses features like auditing and some policy
+(see below).
```

Comment:
@liggitt @mikedanese - This is the only piece that isn't copied from the scope we discussed. PTAL